Client/src/Components/AboutCSS.css

```diff
@@ -18,6 +18,15 @@
     font-weight: 700;
     letter-spacing: -7px;
 }
+
+#about-title{
+    font-family: "League Spartan", sans-serif;
+    background: url("../assets/bg1.webp") 50% 50%;
```

Comment:
Consider using CSS variables to manage repeated values like background URLs, which can reduce code duplication and ease maintenance. [important]